CODEOWNERS

```diff
@@ -48,13 +48,14 @@ Kconfig*                                  @SebastianBoe
 /include/event_manager.rst                @b-gent
 /include/profiler.rst                     @b-gent
 /lib/bin/                                 @rlubos @lemrey
-/lib/at_cmd_parser/                       @rlubos
+/lib/at_cmd_parser/                       @rlubos @metc
```

Comment:
Please split this into a separate commit.

Reply:
Do I need a new commit because of the changes in line 51 or because of line 58?
In this PR, all the changes have been done in one commit: https://github.com/NordicPlayground/fw-nrfconnect-nrf/commit/67ded39d123cf6b3b7ae14150d180842bf0cb721#diff-5daf978e60e03b3975b485d5f1b449dc